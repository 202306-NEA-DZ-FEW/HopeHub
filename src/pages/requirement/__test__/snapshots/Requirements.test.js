import renderer from "react-test-renderer";

import Requirement from "../..";

it("renders correctly", () => {
    const tree = renderer.create(<Requirement />).toJSON();
    expect(tree).toMatchSnapshot();
});
