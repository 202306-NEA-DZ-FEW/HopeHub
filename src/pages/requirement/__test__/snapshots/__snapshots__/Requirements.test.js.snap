// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  dir="ltr"
>
  <div
    className="navbar h-8 sticky top-0 z-10 bg-white backdrop-filter backdrop-blur-lg bg-opacity-30 border-b-slate-400"
  >
    <div
      className="navbar-start"
    >
      <div
        className="dropdown"
      >
        <label
          className="btn btn-ghost md:pl-6 lg:hidden"
          tabIndex={0}
        >
          <svg
            className="h-5 w-5"
            fill="none"
            stroke="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M4 6h16M4 12h8m-8 6h16"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
            />
          </svg>
        </label>
        <ul
          className="menu menu-sm dropdown-content -ml-2 p-2 shadow w-28 mt-2 bg-Primary"
          tabIndex={0}
        >
          <li
            className="text-Accent font-bold underline font-poppins"
          >
            <a
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Home
            </a>
          </li>
          <li
            className="text-NeutralBlack font-semibold font-poppins"
          >
            <a
              href="/blogs"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Blogs
            </a>
          </li>
          <li
            className="text-NeutralBlack font-semibold font-poppins"
          >
            <a
              href="/about"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              About
            </a>
          </li>
          <li
            className="text-NeutralBlack font-semibold font-poppins"
          >
            <a
              href="/contact"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Contact
            </a>
          </li>
        </ul>
      </div>
      <div
        className="logo-container lg:px-6"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <img
            alt="Logo"
            data-nimg="1"
            decoding="async"
            height={60}
            loading="lazy"
            onError={[Function]}
            onLoad={[Function]}
            src="/_next/image?url=%2Fimg.jpg&w=256&q=75"
            srcSet="/_next/image?url=%2Fimg.jpg&w=128&q=75 1x, /_next/image?url=%2Fimg.jpg&w=256&q=75 2x"
            style={
              Object {
                "color": "transparent",
              }
            }
            width={120}
          />
        </a>
      </div>
    </div>
    <div
      className="flex justify-between ml-auto"
    >
      <div
        className="navbar-center hidden lg:flex"
      >
        <ul
          className="menu menu-horizontal"
        >
          <li
            className="text-Accent font-bold underline font-poppins text-base tracking-wider"
          >
            <a
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Home
            </a>
          </li>
          <li
            className="text-NeutralBlack font-semibold font-poppins text-base tracking-wider "
          >
            <a
              href="/blogs"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Blogs
            </a>
          </li>
          <li
            className="text-NeutralBlack font-semibold font-poppins text-base tracking-wider "
          >
            <a
              href="/about"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              About
            </a>
          </li>
          <li
            className="text-NeutralBlack font-semibold font-poppins text-base tracking-wider"
          >
            <a
              href="/contact"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
            >
              Contact
            </a>
          </li>
        </ul>
      </div>
      <button
        className="button-container w-28 h-9 mx-4 md:mr-8 md:w-40 bg-Accent hover:bg-Primary rounded-md"
      >
        <a
          className="text-base tracking-wider text-NeutralBlack font-extrabold font-poppins"
          href="/login"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "textTransform": "capitalize",
            }
          }
        >
          Log In
        </a>
      </button>
    </div>
  </div>
  <div
    className="mt-8"
  >
    <div
      className="bg-NeutralWhite min-w-screen min-h-screen flex flex-row-reverse"
    >
      <img
        alt=""
        className="hidden lg:block px-20 pb-[32rem] "
        data-nimg="1"
        decoding="async"
        height={400}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="/_next/image?url=%2Fimg.jpg&w=1200&q=75"
        srcSet="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1200&q=75 2x"
        style={
          Object {
            "color": "transparent",
          }
        }
        width={550}
      />
      <div
        className="w-full h-full lg:w-[70%] px-8 lg:px-20 bg-NeutralWhite pt-4 pb-12 flex flex-col group"
      >
        <h1
          className="mb-5 font-poppins font-bold tracking-wider text-NeutralBlack capitalize text-2xl lg:text-4xl leading-normal"
        >
          Why work with HopeHub?
        </h1>
        <div
          className="Capitalize text-NeutralBlack text-lg lg:text-2xl font-semibold font- mb-2"
        >
          Reliable Income
        </div>
        <p
          className="text-NeutralBlack text-lg font-poppins text-justify mb-6"
        >
          Over 10,000 people sign up on HopeHub every day looking for a counselor to help with life challenges. HopeHub can be your main source of income (full time) or a supplement to your current work.
        </p>
        <div
          className="Capitalize text-NeutralBlack text-lg lg:text-2xl font-semibold font-poppins mb-2"
        >
          Focus on Counseling
        </div>
        <p
          className="text-NeutralBlack text-lg font-poppins text-justify mb-6"
        >
          No need to worry about costs from acquiring clients, billing, support, or operations. Let us handle the fees and paperwork so you can focus on what you do best!
        </p>
        <h1
          className="mb-5 pt-4 font-poppins font-bold tracking-wider text-NeutralBlack capitalize text-2xl lg:text-4xl"
        >
          Requirements
        </h1>
        <ul
          className="text-NeutralBlack text-lg font-poppins mb-6"
        >
          <li
            className="mb-2"
          >
            ● 
            Licensed by a State Board to provide counseling
          </li>
          <li
            className="mb-2"
          >
            ● 
            Experience in counseling for adults, couples, and/or teens
          </li>
          <li
            className="mb-2"
          >
             
            ● 
            Excellent writing skills
          </li>
          <li
            className="mb-2"
          >
             
            ● 
            Reliable Internet connection
          </li>
          <li
            className="mb-2"
          >
             
            ● 
            Currently residing in the US
          </li>
          <li
            className="mb-2"
          >
            ● 
            Licensed by a State Board to provide counseling
          </li>
        </ul>
         
        <button
          className=" mt-6 self-end w-28 h-10 rounded-md text-base font-poppins font-regular bg-Accent text-NeutralBlack group-hover:bg-[#879AB8] group-hover:text-NeutralWhite group-hover:scale-105 duration-500"
        >
          Next
        </button>
      </div>
    </div>
  </div>
  <footer
    className="footer px-10 py-5 bg-[#BFDFDC] text-base-content flex flex-col md:flex-row lg:flex-row justify-between items-center"
  >
    <form>
      <header
        className=" font-bold text-xl mb-[-15px]"
      >
        footer_msg_1
      </header>
      <fieldset
        className="form-control w-80 mt-2"
      >
        <label
          className="label"
        >
          <span
            className="label-text text-[#718096]"
          >
            footer_msg_2
          </span>
        </label>
        <div
          className="relative flex flex-row"
        >
          <input
            className="input input-bordered border-[#718096] border-solid pr-16 outline-none focus:outline-none"
            placeholder="footer_msg_3"
            type="text"
          />
          <span
            className=" w-20 ml-[-1rem]"
          >
            <svg
              fill="none"
              height="auto"
              viewBox="0 0 100 100"
              width="100%"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M1 1H54C56.7614 1 59 3.23858 59 6V54C59 56.7614 56.7614 59 54 59H1V1Z"
                fill="#99B4DF"
                stroke="#718096"
                stroke-width="2"
              />
              <path
                d="M29.6327 30.3673L25.0421 32.5401C24.5997 32.7861 24.053 32.7438 23.6538 32.4325L17.4862 27.7498C16.6922 27.1308 16.9078 25.8769 17.8628 25.5585L40.3377 18.0669C41.3237 17.7382 42.2618 18.6763 41.9331 19.6623L34.4415 42.1372C34.1231 43.0922 32.8691 43.3078 32.2502 42.5138L27.5675 36.3462C27.2562 35.947 27.2139 35.4003 27.4599 34.9579L29.6327 30.3673Z"
                fill="#1A1A1A"
              />
            </svg>
          </span>
        </div>
      </fieldset>
    </form>
    <div
      className=" items-center flex flex-col justify-center h-full"
    >
      <nav
        className="footer-title flex gap-8"
      >
        <a
          className=" "
          href="../Home"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          Home
        </a>
        <a
          className=" "
          href="../Blogs"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          Blogs
        </a>
        <a
          className=" "
          href="../About"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          About
        </a>
        <a
          className=" "
          href="../Contact"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          Contact
        </a>
      </nav>
      <div
        className="grid grid-flow-col gap-4"
      >
        <a
          href=""
        >
          <svg
            fill="none"
            height="35"
            viewBox="0 0 35 35"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M29.2578 4.62682C30.9347 4.28241 32.5884 3.39473 32.5884 3.39473C33.3256 3.04856 33.6012 3.31906 33.2037 4.01651C33.2037 4.01651 32.2112 6.07413 30.6949 7.02984C32.1797 6.84341 33.52 6.29924 33.52 6.29924C34.2926 6.0365 34.5007 6.66055 33.9952 7.29167C33.9952 7.29167 32.6842 9.11819 31.25 10.2083C31.2624 10.538 31.2682 10.4798 31.2682 10.8164C31.2682 22.4922 25.3761 32.8125 10.3789 32.8125C4.74279 32.8125 1.72544 30.5762 1.72544 30.5762C0.269777 29.8354 0.435023 28.9706 2.05229 28.7287C2.05229 28.7287 7.48179 28.2115 9.99381 26.1372C-4.37503 18.9583 1.48563 5.05932 1.48563 5.05932C1.54196 4.25827 2.09591 4.05424 2.68917 4.61642C2.68917 4.61642 9.15627 11.3561 16.7492 11.6789C16.6231 11.1127 16.5601 10.525 16.5601 9.91857C16.5601 5.64926 19.8479 2.1875 23.8993 2.1875C26.0129 2.1875 27.922 3.12555 29.2578 4.62682Z"
              fill="#1A1A1A"
            />
          </svg>
        </a>
        <a
          href=""
        >
          <svg
            fill="none"
            height="35"
            viewBox="0 0 35 35"
            width="35"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1.45837 17.5389C1.45837 8.65781 8.64047 1.45825 17.5 1.45825C26.3596 1.45825 33.5417 8.65781 33.5417 17.5389C33.5417 25.8821 27.2033 32.7413 19.0902 33.5416C19.1336 33.5339 19.1769 33.5258 19.2202 33.517V22.155H23.3908L24.0148 17.3012H19.2202V14.2023C19.2202 12.7969 19.6093 11.8395 21.6199 11.8395L24.1837 11.8382V7.49707C23.7402 7.43772 22.2185 7.30577 20.4476 7.30577C16.7511 7.30577 14.22 9.56756 14.22 13.7218V17.3012H10.0388V22.155H14.22V33.2832C6.93355 31.7655 1.45837 25.2932 1.45837 17.5389Z"
              fill="#1A1A1A"
            />
          </svg>
        </a>
        <a
          href=""
        >
          <svg
            fill="none"
            height="33"
            viewBox="0 0 33 33"
            width="33"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M16.7259 13.3233V18.6751C16.7259 19.2279 17.1737 19.6759 17.7266 19.6752C19.597 19.6727 23.6064 19.6678 25.5202 19.6678C24.1419 23.787 21.9994 26.0297 16.7259 26.0297C11.3888 26.0297 7.22328 21.763 7.22328 16.4993C7.22328 11.2368 11.3888 6.97012 16.7259 6.97012C19.5473 6.97012 21.3699 7.9484 23.0415 9.3115C24.3792 7.99213 24.2675 7.80347 27.6716 4.6325C24.7816 2.03874 20.9409 0.458252 16.7259 0.458252C7.74238 0.458252 0.458374 7.6398 0.458374 16.4993C0.458374 25.3588 7.74238 32.5416 16.7259 32.5416C30.1541 32.5416 33.4363 21.0109 32.3486 13.3233H16.7259Z"
              fill="#1A1A1A"
            />
          </svg>
        </a>
      </div>
    </div>
  </footer>
</div>
`;
